helm-charts/falcon-sensor/templates/clusterrole.yaml

```diff
@@ -16,6 +16,22 @@ rules:
   - ""
   resources:
   - secrets
+  {{- if and .Values.node.enabled .Values.node.cdpRolesEnabled }}
```

Comment:
* I am not able to comment on content that is not part of the diff
* https://github.com/CrowdStrike/falcon-helm/pull/340/files#diff-8b1b64050a16b4744833a69c1dd56929ef1e6ef30cbc080ce0d4018c94a46f28R11
* https://github.com/CrowdStrike/falcon-helm/pull/340/files#diff-6ce3e7fab19d7aa64a73efef8b41d75116dc829eb636e3e22cfce3984e99d308L11
* `app.kubernetes.io/component` should be updated to `all_sensor`.
   * Not in the scope of this PR
      * We should replace `kernel_sensor` as `node_sensor`. We never updated the component after introducing support for `bpf`. Adding @gpontejos

Reply:
Checked other resources that we create; `all_sensor` is applicable, only if we use the same resource name for multiple sensor types. In this case we create resource using the variable `"falcon-sensor.fullname"` and that is unique per deployment. So we should add
```
{{ if .Values.container.enabled }}
app.kubernetes.io/component: "container_sensor"
{{ else if (and .Values.node.enabled .Values.node.cdpRolesEnabled)}}
app.kubernetes.io/component: "kernel_sensor"
{{ end }}
```